client/src/components/Notify.re

```diff
@@ -1,6 +1,7 @@
-Modules.require("./Notify.css");
+Modules.import("./Notify.css");
```

Comment:
Please undo all the changes from `require` to `import`. They will break the app. We want `require`.

Reply:
Returned back.